src/SFA.DAS.SharedOuterApi/Infrastructure/HealthCheck/ApprenticeAccountsApiHealthCheck.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Api.Common.Infrastructure;
+using SFA.DAS.SharedOuterApi.Configuration;
+using SFA.DAS.SharedOuterApi.InnerApi.Requests;
+using SFA.DAS.SharedOuterApi.Interfaces;
+
+namespace SFA.DAS.SharedOuterApi.Infrastructure.HealthCheck
+{
+    public class ApprenticeAccountsApiHealthCheck : IHealthCheck
+    {
+        private const string HealthCheckResultDescription = "Apprentice Accounts Api check";
+
+        private readonly IApprenticeCommitmentsApiClient<ApprenticeAccountsApiConfiguration> _apiClient;
```

Comment:
shouldn't this be IApprenticeAccountsApiClient<>

Reply:
good spot!